src/swell/swell.py

```diff
@@ -221,7 +245,7 @@ def test(test):
 
 @swell_driver.command()
 @click.argument('suite', type=click.Choice(("hofx", "3dvar", "ufo_testing")))
-def t1test(suite):
+def t1test(suite: str) -> None:
```

Comment:
We can actually be even stricter with the type hinting here by restricting to specific values, which helps catch a higher level of bug (and provides even better inline documentation):

```suggestion
def t1test(suite: Literal["hofx", "3dvar", "ufo_testing"]) -> None:
```

(Requires `from typing import Literal` above)

Reply:
Changed!